packages/node-build-scripts/package.json

```diff
@@ -11,10 +11,10 @@
     },
     "bin": {
         "assert-package-layout": "./assert-package-layout.mjs",
-        "css-dist": "./css-dist.sh",
+        "css-dist": "./css-dist.mjs",
```

Comment:
the latest Yarn doesn't like running NPM binaries as shell scripts, it tries to run everything in Node.js. the easiest thing to do here is to migrate to JS scripts, even though it's a little more verbose. this also has the benefit of standardizing all the script formats, but it is a small breaking change.